lib/oli_web/live/delivery/remix_section.html.heex

```diff
@@ -28,7 +28,7 @@
             <p>There's nothing here.</p>
           </div>
         <% end %>
-        <%= for {node, index} <- build_entries(@active.children, assigns, @dragging) do %>
+        <%= for {node, index} <- Enum.with_index(@active.children) |> Enum.filter(fn {c, _i} -> c.uuid != @dragging end) do %>
```

Comment:
Can you move out of template and into function?

Reply:
Definitely 💯